data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -16,23 +16,26 @@ if ! release=$( podman inspect {{.ReleaseImage}} -f '{{"{{"}} index .RepoDigests
 	release="{{.ReleaseImage}}"
 fi
 
-MACHINE_CONFIG_OPERATOR_IMAGE=$(podman run --quiet --rm ${release} image machine-config-operator)
-MACHINE_CONFIG_OSCONTENT=$(podman run --quiet --rm ${release} image machine-os-content)
-MACHINE_CONFIG_ETCD_IMAGE=$(podman run --quiet --rm ${release} image etcd)
-MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE=$(podman run --quiet --rm ${release} image kube-client-agent)
-MACHINE_CONFIG_INFRA_IMAGE=$(podman run --quiet --rm ${release} image pod)
+bootkube_podman_run="podman run --quiet --net=host"
+image_for="podman run --quiet --rm --net=none ${release} image"
```

Comment:
ShellCheck doesn't seem to have a problem with your approach, but I'd be more comfortable with shell functions:

```sh
bootkube_podman_run() {
  podman run --quiet --net=host "${@}"
}

image_for() {
  RELEASE="${1}"
  podman run --quiet --rm --net=none "${RELEASE}" image
}

MACHINE_CONFIG_OPERATOR_IMAGE=$(image_for machine-config-operator)
...
  bootkube_podman_run \
    --volume "$PWD:/assets:z" \
...
```

Reply:
hah, that's because we don't shellcheck bootkube.sh.